### DISCUSSION TITLE
How do i display a snapshot of a zone event in HA?

### DISCUSSION THREAD

**Discussion by R5fan** [blakeblackshear/frigate#1236] [Ask A Question]
My camera's track various objects and create snapshots for all of them. However, most of them are not interesting (cars on the street, birds attacking the camera). I still like to keep those, because they are helpful for debugging and some frivolous things like counting cars.  I am however, mostly interested in objects entering certain zones, like my driveway, for which I also keep clips.

The HA integration creates a camera for every snapshot of a tracked label and for every camera, but not for zones, much less a combination of criteria. I would like to see the last "person on driveway"  and "last car on driveway" or at least the "last object on driveway" (even if its a bird) on my dashboard. Is there any way to do this currently?

**Comment by blakeblackshear** [comment]
The best way to accomplish this with current functionality is to use the `required_zones` under the camera specific `mqtt` config section. This will require an object to have entered a zone before publishing the image to mqtt. This should accomplish "last object on driveway".

**Comment by R5fan** [comment]
But then I lose the snapshots and events for everything else, right? If so, Id rather not. As a (crude) work around, I suppose I could duplicate the cameras and have one configured with and one without a "required_zones" ? But that would presumably also double bandwidth and processing requirements...

**Comment by blakeblackshear** [comment]
You wouldn't loose snapshots and events for other objects. This just defines the requirements for publishing the image to mqtt. It does not impact anything else.
